Model/Config.php

```diff
@@ -367,6 +367,17 @@ public function getBalanceIframeUrl($scope = ScopeInterface::SCOPE_STORE, $store
             ? self::BALANCEPAY_IFRAME_SANDBOX_URL : self::BALANCEPAY_IFRAME_LIVE_URL);
     }
 
+    /**
+     * @param $scope
+     * @param $storeId
+     * @return string[]
+     * @throws \Magento\Framework\Exception\NoSuchEntityException
+     */
+    public function getAllowedCustomerGroups($scope = ScopeInterface::SCOPE_STORE, $storeId = null)
+    {
+        return explode(',', $this->getConfigValue('allowed_customer_groups', $scope, $storeId));
```

Comment:
What this function will return if `$this->getConfigValue('allowed_customer_groups', $scope, $storeId)` return empty string? (if that's even possible)